lib/helper.rb

```diff
@@ -0,0 +1,23 @@
+# This is a helper class to run helper methods
+# This is a class which all of the methods about user interface will be in
+class Helper
+  def initialize; end
+
+  def self.validate_player_name(player_1_name, player_2_name = 'not set')
+    return player_1_name if player_2_name == 'not set'
+
+    until player_2_name != player_1_name
+      puts 'Be careful! Player 2 name must be different that Player 1 name'
```

Comment:
You don't have to write the code twice. Instead of using `puts` you can return a string and use it in `main.rb`

Reply:
Thank you for your review,  I updated them.